config.example.php

```diff
@@ -44,17 +44,18 @@
     ],
 
     'filters' => [
-        'include' => [
-            // if empty include all by default
+        'include' => [                                          // if empty include all by default
```

Comment:
From my point of view easier to understand and to avoid [such discussions](https://www.ip-phone-forum.de/threads/carddav-to-fritzbox-2-0-neue-version-verf%C3%BCgbar.298117/post-2334259).

Reply:
Welches renaming? Ich versteh den Kommentar nicht. Willst du hier was ändern?